databases/relational-databases.md

```diff
@@ -55,10 +55,59 @@ To understand, query, and insert information into a relational database, a techn
 - [SQL Fiddle](http://sqlfiddle.com)
 
 ### Lesson
+
+- The specific problems a database solves
+  - Persistence of information
+  - Centralized information
+  - Search stored information
+  - Form relationships between sets of information
+- Users of databases
+  - Front end people should have concepts and write very basic SQL
+  - Back end people spend a large amount of time with databases
+- Structure of a relational database; tables with columns and references to other tables
+- Few basic column types
+  - `varchar`
+  - `text`
+  - `integer`
+  - `primary key`, `auto increment`
+    - exact type varies depending on database type
+- General SQL statement pattern (verb-subject-optional modifiers)
+- The four basic SQL verbs and their use
+  - `SELECT`
+    - without modifiers
+    - selecting specific columns: \*, specific columns, AS column aliasing
+    - modifier constraints:
+      - `WHERE`
+      - `ORDER BY`
+      - `LIMIT`
+      - `JOIN`
+        - Should only cover `LEFT INNER/NATURAL/JOINS` with resources pointing to other types
+  - `INSERT INTO`
+    - without modifiers (entire table)
+    - with modifier constraints (same as `SELECT`)
+    - from another table `(INSERT INTO a SELECT x, y, z from b)`
+    - from another table with constraints `(INSERT INTO a SELECT x, y, z FROM b WHERE x = 'a thing')`
+    - Would be good to demonstrate type mismatch (`INSERT` '1' into an integer column), but if using MySQL
+      to teach this doesn't work as MySQL automatically type casts depending on server config
+  - `UPDATE...SET`
+    - without modifiers (entire table)
+    - with modifier constraints (same as `SELECT`)
+  - `DELETE FROM`
+    - without modifiers (entire table)
+    - with modifier constraints (same as `SELECT`)
+- DDL to create tables
+  - `CREATE TABLE`
+- Bonus topic for the advanced: foreign key constraints
+- Explain different databases with slightly different dialects (MySQL - PostgreSQL)
```

Comment:
Do these 2 lines beginning with ‘explain ‘ ever get explained in this outline? If yes, these need to be rephrased. If no, please provide more information.